superduperdb/backends/mongodb/query.py

```diff
@@ -137,7 +137,7 @@ def outputs(self, **kwargs):
         return Find(
             name=self.name,
             type=self.type,
-            args=self.args,
+            args=args,
             kwargs=self.kwargs,
```

Comment:
Does kwargs have the same problem？

Reply:
no self.kwargs are not mutated in this function